src/index.tsx

```diff
@@ -503,12 +505,12 @@ function ReactSimplyCarousel({
           isMissingTransitionEndRef.current = true;
         }
 
-        itemsListRef.current!.style.transform = nextItemsListTransform;
+        itemsListRef.current!.style.transform = nextItemsListTransform as any;
 
         onRequestChange(newActiveSlideIndex, {
-          visibleSlides: nextVisibleSlides,
-          isFirstSlideVisible: nextIsFirstSlideVisible,
-          isLastSlideVisible: nextIsLastSlideVisible,
+          visibleSlides: nextVisibleSlides as any,
```

Comment:
Why `any`?

Reply:
Cause my VSCode is complaining:

<img width="649" alt="Screenshot 2023-05-03 at 20 13 26" src="https://user-images.githubusercontent.com/25827414/236007163-967657ad-32ed-4063-aaa4-ab36fc937d40.png">

Unfortunately I can't afford to dive deep to fix typings. Might as well fallback to an empty string?..
```js
itemsListRef.current!.style.transform = nextItemsListTransform || '';
```

Reply:
On the other hand... check out my fresh commit [e168e94](https://github.com/vadymshymko/react-simply-carousel/pull/146/commits/e168e947e37826a95713d1ff62edebc12c66e652)